webpages/js/papers.js

```diff
@@ -1355,16 +1425,16 @@
     setTimeout(doMoveColumn, 0, this);
   }
 
+// hedgehog - Maybe we need to fix this shit
```

Comment:
Mostly certain it was because we put in .index (in place of colid) and we didn't check it all worked fine. I am in the middle of some work to move experiments and papers, so i'll give all the movement controls a poke and if I can't find anything I'll remove it.